wireguard-go/README.md

```diff
@@ -0,0 +1,39 @@
+This directory contains to patches that can be applied to wireguard-go to leverage wolfCrypt as a crypto engine.
```

Comment:
Should probably specify host machine requirements. Linux? MacOS?

Reply:
Good suggestion, done.